.github/workflows/plugins-test.0.yaml

```diff
@@ -65,7 +65,7 @@ jobs:
           - cassandra-java-driver-3.x-scenario
           - customize-scenario
           - correlation-autotag-scenario
-          - dubbo-2.5.x-scenario
+          - dubbo-scenario
```

Comment:
Why do you remove 2.5? I think 2.5 is an Alibaba version, and 2.7 is an Apache version. They are different.

Reply:
The Alibaba version 2.5.x and 2.6.x are in `dubbo-scenario`, the Apache version 2.7.x + is in `dubbo-2.7.x-scenario`.

Why rename? Because `dubbo-2.5.x-scenario` contains 2.5.x and 2.6.x.

Reply:
Let's keep them, or rename to alibaba-dubbo-2.5

We usually keep the minimum version in plugin name only, not all.